src/components/DpAddonWrapper/DpAddonWrapper.vue

```diff
@@ -0,0 +1,87 @@
+<template>
```

Comment:
I see the need to move DpAddonWrapper.vue into demosplan-ui... moving it to [demosplan-js-addon](https://github.com/demos-europe/demosplan-js-addon) would be more appropriate but as that repo does not provide a frontend build as of now i understand the reasoning.

However, as we plan to use demosplan-ui in a wider context, i feel that we must communicate the use case here, ecactly. Thats why i'd suggest to have a DpAddonWrapper.stories.mdx which explains the use case and gives a html example for how to use the component. I know that this is all documented within demosplan-core addon_documentation_frontend.md... maybe all we need is cross-linking to that documentation?

Reply:
Can somebody explain the need of moving this file? I don't get the reason behind it.

Reply:
Well the currently developed addon needs to be implemented in the recommendation modal tabs but the recommendation modal as is is also an addon. Hence the addon needs to load an addon which is not a big issue from technically point of view but we need the wrapper to be accessible in the addon too. Hence I thought having the wrapper as part of demosplan-ui would solve this issue.

Reply:
But from an architektural point of view: do we want that? It sound quite bug-prone to me.

Reply:
For refs, we decided it would be better to refactor the recommendation modal in its corresponding addon to be a tab too and provide a generic modal in the core which loads both addons.